weaviate/collections/queries/base.py

```diff
@@ -143,13 +147,12 @@ def __deserialize_primitive(self, value: Any, type_value: Any) -> Any:
     def __parse_nonref_properties_result(
         self,
         properties: Union[search_get_pb2.PropertiesResult, base_pb2.ObjectPropertiesValue],
-        type_: Optional[Any],
+        type_hints: dict,
```

Comment:
type hints for dict?

Reply:
it could be anything, would `Dict[str, Any]` be preferable?

Reply:
if we know that the keys are always strings, yes(same at the other places)